Testing/ORKTest/ORKTest/ORKESerialization.m

```diff
@@ -833,6 +833,12 @@ @implementation ORKESerializer
            PROPERTY(outputVolume, NSNumber, NSObject, NO, nil, nil),
            PROPERTY(samples, ORKToneAudiometrySample, NSArray, NO, nil, nil),
            })),
+   ENTRY(ORKReactionTimeResult,
+         nil,
+         (@{
+            PROPERTY(timestamp, NSNumber, NSObject, NO, nil, nil),
+            PROPERTY(fileResult, ORKResult, NSObject, NO, nil, nil)
```

Comment:
Should `writeAfterInit` be `YES`? If that's the case, the issue affects most of the result subclasses.